.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# automatically requests pull request reviews for files matching the given pattern; the last match takes precendence
+
+*       @spacetelescope/stsci.stimage-maintainers
```

```suggestion
*       @spacetelescope/stsci-stimage-maintainers 
```
Never mind, it looks like the name specifier just uses a hyphen instead.